src/java.base/unix/classes/sun/nio/fs/UnixNativeDispatcher.java

```diff
@@ -391,6 +391,22 @@ static void lutimes(UnixPath path, long times0, long times1)
     private static native void lutimes0(long pathAddress, long times0, long times1)
         throws UnixException;
 
+    /**
+     * utimensat(int fd, const char* path,
+     *           const struct timeval times[2], int flags)
+     *
+     * We hard code fd to FD_ATCWD and flags to AT_SYMLINK_NOFOLLOW.
+     */
+    static void utimensat(UnixPath path, long times0, long times1)
```

Comment:
The *at methods that are supposed to map 1-1 to the similarly named sys calls so I'm a bit uncomfortable having this one work differently and hardcode dirfd  to FD_ATCWD. I wonder if we could generate a value for FD_ATCWD in UnixConstants.java.template so that the use site would more closely match the sys call.

Reply:
> I wonder if we could generate a value for FD_ATCWD in UnixConstants.java.template so that the use site would more closely match the sys call.

Sure, no problem.

Reply:
So changed in 26feef6.